frontend/src/views/c-summary.vue

```diff
@@ -708,7 +708,7 @@ export default {
         const filteredFileTypes = this.getFilteredFileTypes(result);
         this.updateCommitResultWithFileTypes(result, filteredFileTypes);
         return result;
-      }).filter((result) => Object.values(result.fileTypesAndContributionMap).length > 0);
```

Comment:
Great work. Can I ask why this filter is removed? According to the behavior of the commits panel, the merge commits without any addition or deletion should only be shown when every file type is checked. If certain file types are unchecked, then these need to be hidden.

Reply:
Thanks! I have changed the behavior of the Ramp Chart to be similar to the Commits Panel.